src/components/backgroundImage.js

```diff
@@ -0,0 +1,12 @@
+import styled from '@emotion/styled';
+import { Box } from 'theme-ui';
+
+const BackgroundImage = styled(Box)`
```

Comment:
This component should not be `BackgroundImage`. Ideally, it should just be `Background` accepting color or url, size, and repeat props.